plenum/persistence/leveldb_hash_store.py

```diff
@@ -78,6 +79,7 @@ def open(self):
         self.nodesDb = KeyValueStorageLeveldb(self.dataDir, self.nodes_db_name)
         self.leavesDb = KeyValueStorageLeveldb(
             self.dataDir, self.leaves_db_name)
+        self._leafCount = self.leavesDb.size
```

Comment:
I think we should increment it in `writeLeaf` method (currently `self.leafCount` is used there, which is a different field)